mpp/shmem.h.in

```diff
@@ -18,6 +18,10 @@
 
 #include <stddef.h>
 #include <complex.h>
```

Comment:
The relevant StackOverflow post: http://stackoverflow.com/questions/22115996/complex-type-with-c-linkage-in-c11

I'll write up some test cases to see what nonsense you have to do to use complex in C++.